packages/cli/src/command.js

```diff
@@ -36,7 +36,7 @@ export default class NuxtCommand extends Hookable {
   }
 
   async run () {
-    await this.callHook('setup', {
+    await this.callHook('run:before', {
```

Comment:
This will be a **breaking** change for ts module users that only upgrade their nuxt. Is this OK? (I think for an internal hook like this setup name is just fine) 

so still merge this? :)

Reply:
@pi0 Well current Nuxt release has `setup` hook not called at the really wanted place, and Nuxt TS with hooks is only draft PR.

We talked about **breaking** change indeed for people upgrading module but not Nuxt, but we said we want to force user to use last hooky Nuxt version , so all good no ? 😁

Reply:
Yep! I thought TS module was released with setup hook. So seems good!